FileAPI/blob/Blob-in-worker.worker.js

```diff
@@ -0,0 +1,14 @@
+importScripts("/resources/testharness.js");
+
+async_test(function() {
+  var data = "TEST";
+  var blob = new Blob([data], {type: "text/plain"});
+  var reader = new FileReader();
+  reader.onload = this.step_func_done(function() {
+    assert_equals(reader.result, data);
+  });
+  reader.onerror = this.unreached_func("Unexpected error event");
+  reader.readAsText(blob);
+}, "Create Blob in Worker");
+
+done();
```

Comment:
This will need to be moved inside the `onload` handler, or else the test will finish immediately.